### PR TITLE
Re-style the popup.

### LINKED ISSUE #9: New design for panel
Hi all,

This is the design for the current panel: 

image

- It doesn't really conform with out other panels in Firefox looks.
- The example image is too distracting, and isn't related to the page.
- The icon in the URL bar has a lot going on in it.
- I think the URL is irrelevant to the user...especially on super complex URLs, I don't think it adds much context.
- The note field is too big as well.

Here's an updated design: 

image

- I made a couple text changes, but really someone from Context Strategy should give it a look over.
- I remove the URL, and instead added the favicon of the page so the user knows it's related to the tab they are on
- I shrunk the notes field and added a clear report button.

Notes:

- Clicking off this panel should close it immediately.
- Once a user clicks the report button, we have two options:

Option A: Just close the panel and pulse the icon once in the URL bar (see how the DRM button pulses the first time you play DRM context).
image

Or Option B: Shrink the panel and show a tiny "Thanks!" That disappears in 2 seconds, or if the user clicks off of it.
image

### PR DESCRIPTION
**Pull request by bwinton**
It might need more work, but this seems like a decent place to stop and get a review. 🙂

* Use browser-styles.
* Move the attribution to the options.
* Just close the popup when we report it.
* Move things around a lot.
* Other stuff I forgot.

Fixes #9, #7, #5, #4, #3.

### CODE REVIEW

File: extension/manifest.json [hunk 1]
```diff
@@ -6,26 +6,32 @@
...
   
     "permissions": [
       "activeTab",
       "tabs"
     ],
   
-    "browser_action": {
-      "default_icon": "icons/icon-64.png",
+    "page_action": {
+      "browser_style": true,
```
Comment (ehsan) [inline extension/manifest.json]: This also causes the manifest to not load in Chrome.

File: extension/page/report.html [hunk 2]
```diff
@@ -1,30 +1,29 @@
...
     <body>
-        <div id="wrapper">
-            <header>
-                <h3>Are you tired of seeing in-page popups like this?</h3>
-                <div><img src="popup.jpg" width="335" height="224"></div>
-                Mozilla is experimenting with a popup blocker to dismiss them automatically, and are curating a dataset for it.
-                Help Mozilla by reporting pages that show pop-ups like this.
-            </header>
+        <div class="panel-section panel-section-header">
+            <img class="icon-section-header" src="popup.jpg">
```
Comment (ehsan) [inline extension/page/report.html]: Is it possible to enlarge this icon a bit?  Currently the image is hardly visible.  If you think the image doesn't convey anything useful, I'm also happy to remove it.  But I think having it too small doesn't make all that much sense, what do you think?

(Happy to trust your final say on this!)

**Review by ehsan — COMMENTED** [review]
Overall this looks really great!  I have one comment on the design, but I'm inclined to release a new update for the Firefox version of the extension.

Is there any chance you would be interested to look into Chrome compatibility as well?  :-)  Around 1/6th of the userbase of this extension is currently on Chrome...

(I'd also be happy to merge the PR and do the rest of the work myself.)

File: extension/page/report.html (hunk 2 above)
Comment (bwinton) [reply extension/page/report.html]: I think @sevaan wanted to use a different icon here, but I don't have the assets for it (yet 😉)…
I'll update the PR once I get them!

File: extension/manifest.json [hunk 3]
```diff
@@ -6,26 +6,32 @@
...
-      "16": "icons/icon-16.png",
-      "24": "icons/icon-24.png",
-      "32": "icons/icon-32.png",
-      "64": "icons/icon-64.png",
-      "128": "icons/icon-128.png"
+      "16": "icons/icon.svg",
+      "24": "icons/icon.svg",
+      "32": "icons/icon.svg",
+      "64": "icons/icon.svg",
+      "128": "icons/icon.svg"
```
[the remark this answers is not in the document]
Comment (bwinton) [reply extension/manifest.json]: Nah, I'll just go back to using the pngs. :)

[the remark this answers is not in the document]
Comment (ehsan) [reply extension/manifest.json]: Good question.  This is the first time I come across this question, so I decided to search how others have done this.

I came across [this article](https://medium.com/@yorkxin/things-i-learned-from-migrating-a-chrome-extension-to-firefox-using-webextensions-975474d4fa77) which explains how to do this [using webpack](https://github.com/chitsaou/copy-as-markdown/blob/bf21d96e33b9bc5afcc3f4546aa9e392cfd012a9/webpack.config.js#L36) and two separate manifest files.  How does that sound?

[the remark this answers is not in the document]
Comment (bwinton) [reply extension/manifest.json]: I'm not a huge fan of webpack, but other than that it seems reasonable… 🙂

[the remark this answers is not in the document]
Comment (ehsan) [reply extension/manifest.json]: I'm open to anything else really, it's my first time using any of these tools, so feel free to pick anything else!

[the remark this answers is not in the document]
Comment (ehsan) [reply extension/manifest.json]: (FWIW I would be equally happy with a shell script!)

**Comment by ehsan** [comment]
Thanks a lot!

**Pull request merged**